go/validationlib/basic_check_test.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2018 The Grafeas Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package validationlib
+
+import "testing"
+
+func TestIsAlpha(t *testing.T) {
```

Comment:
These should be table tests:
https://github.com/golang/go/wiki/TableDrivenTests

Also TAL at how the existing validation tests are done:
https://github.com/grafeas/grafeas/blob/master/go/v1/api/validators/image/image_test.go

Reply:
done, thx